TRAINING_IN_DETAIL.rst

```diff
@@ -0,0 +1,285 @@
+.. _header-n795:
+
+Using SageMaker Containers for SageMaker Training
+=================================================
+
+This document discusses in detail how to use SageMaker Containers for training. 
+
+.. contents::
+
+.. _header-n860:
+
+
+Building and training your own algorithm container using SageMaker Containers
+-----------------------------------------------------------------------------
+
+SageMaker Containers makes easier the process to Build your own container (BYOC). In this scenario, ``SAGEMAKER_PROGRAM``, containing the name of the
+entry point script located under ``/opt/ml/code`` folder is the only
+environment variable required. Alternatively, a hyperparameter named
+``sagemaker_program`` can be used. The workflow to train a BYOC
+container is a follows:
+
+.. figure:: byoc-workflow.png
+   :alt: byoc training workflow
```

Comment:
"entry point" appears as one word a couple times in the diagram

Reply:
Fixed the diagrams.